site/content/en/docs/Guides/access-api.md

```diff
@@ -93,9 +94,17 @@ func main() {
 	// Create a GameServer
 	gs := &agonesv1.GameServer{ObjectMeta: metav1.ObjectMeta{GenerateName: "udp-server", Namespace: "default"},
 		Spec: agonesv1.GameServerSpec{
+			Container: "udp-server",
+			Ports: []agonesv1.GameServerPort{{
+				ContainerPort: 7654,
+				HostPort:      7654,
+				Name:          "gameport",
+				PortPolicy:    agonesv1.Static,
+				Protocol:      corev1.ProtocolUDP,
+			}},
 			Template: corev1.PodTemplateSpec{
 				Spec: corev1.PodSpec{
-					Containers: []corev1.Container{{Name: "udp-server", Image: "{{% example-image %}}"}},
+					Containers: []corev1.Container{{Name: "udp-server", Image: "gcr.io/agones-images/udp-server:0.17"}},
```

Comment:
Why did we replace this? Much easier to keep this in the shortcode?

Reply:
that's right, I also want to add this as ENV variable for a container, not to update image every time we update `udp-server`.

Reply:
Since this is documentation, shouldn't we keep it as the shortcode? Otherwise everywhere else in the documentation we'll point at the latest release, and this will be different - which could be confusing.

Reply:
That's right, I will apply shortcode